Cuphead/Cuphead.asl

```diff
@@ -77,7 +75,9 @@ init
 
 		vars.GetAllLevelsData = (Func<List<dynamic>>)(() =>
 		{
-			var levels = vars.Helper.ReadList<IntPtr>(current.SaveSlot + pd["levelDataManager"], pldm["levelObjects"]);
+			// pldm["levelObjects"] sometimes causes a Sequence contains no matching element error in MonoClass[int]
+			// don't know why, it seems random, but it's always 0x10 anyways
+			var levels = vars.Helper.ReadList<IntPtr>(current.SaveSlot + pd["levelDataManager"], 0x10);
```

Comment:
Replace this with a check on whether the element is contained. Return `false` if not the case.

Reply:
Why return false?

Reply:
The entire point of `TryOnLoad` being a `Func<dynamic, bool>` is so that you can `return false` when something in the func goes wrong and the code can retry earlier.

```cs
if (pd.Fields.Count == 0)
	return false;
```

Reply:
That would return false inside the GetAllLevelsData function and not TryOnLoad, no?

Besides, this error occurs repeatedly and there seems to be no indicator when the ASL Helper actually loads that the field is missing. It still shows with the correct offset in the logs. I'm not sure reloading would fix anything (not something that's easy for me to test as I mentioned above).

Reply:
Oops, it seems I lied. pldm has "No fields." and pldo is "Not found!" apparently. OK, will make this change